grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -210,12 +211,35 @@ class IssueService {
         input.collect { element ->
             if (issue.getProperties().get(element.key) != null && element.key != "fundings") {
                 updatedIssue.(element.key) = element.value
+                updatedIssue.setUpdateDate(new Date())
             }
         }
         updatedIssue.save(flush:true)
         updatedIssue
     }
 
+    void projectApproval(Issue issue) {
+        Boolean projectGeneralDataApproved = false
+        Boolean projectDocumentsApproved = false
+
+        if (issue != null && !issue.getAttachments().isEmpty()) {
+            issue.getNonEmptyExtraProperties()?.each{
```

Comment:
Please try to use filter or find here instead of each

Reply:
Done!